### DISCUSSION TITLE
İbax Formuna Getirilen Referanslar!

### DISCUSSION THREAD

**Discussion by Majezikonline** [IBAX-io/go-ibax#458] [Ideas] [answered]
### Clear and concise description of the problem

İbax site ve forumuna yeni getirilen her üye için +5 - +10 ibax coini verilerek daha fazla üyeye ulaşmasını sağlamak

### Suggested solution

İlerleyen zamanlarda foruma KCY(kimlik, telefon ve selfie)doğrulması getirilerek; her üyeye ait bir referans kodunun olması ve bu referans kodu ile üye çağırıp, yeni getirilen üyenin kcy dogrulamasını geçtikten sonra +5 yada +10 ibax coininin üye çağıran kişiye verilmesi.

### Alternative

Kcy, telefon, eposta ve selfie doğrulaması

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by manaba60** [comment]
Devamı olsun lütfen

**Comment by ibaxio** [comment, marked as answer]
Decentralization and anonymity are the beliefs of the IBAX team, so we do not perform KYC verification, let alone collect user privacy.

We have been approached by development teams for technical support to create a DID (Decentralized Identity) ecosystem.

Thanks you for your support.
